backend/src/main/java/heartbeat/controller/report/dto/request/ReportType.java

```diff
@@ -2,23 +2,18 @@
 
 public enum ReportType {
 
-	METRIC("metric"), PIPELINE("pipeline"), BOARD("board");
+	BOARD("board"), DORA("dora");
```

Comment:
ReportType is defined `MetricType` in main branch

Reply:
We refine the domain